connexion/decorators/response.py

```diff
@@ -56,6 +56,14 @@ def validate_response(self, data, status_code, headers, url):
                 msg = ("Keys in header don't match response specification. "
                        "Difference: {0}").format(pretty_list)
                 raise NonConformingResponseHeaders(message=msg)
+            # validate each of the existing keys.
+            for key, schema in response_definition.get("headers").items():
+                header_validator = ResponseHeaderValidator(schema)
+                for data in headers.getlist(key):
```

Comment:
The `.getlist` method is Flask specific. We are supposed to clear our core code from Flask specific stuff. I would recommend creating a new class `ConnexionHeader` that wraps this implementation. The ConnexionResponse class can instantiate this class when populating it's attribute `self.headers`. So whoever is going to implement a new framework support for Connexion will know they have to also implement a method called `getlist` (or maybe `get_list`?).